### DISCUSSION TITLE
Automatically set mermaid themes to match dark / light

### DISCUSSION THREAD

**Discussion by edavidaja** [quarto-dev/quarto-cli#1521] [Feature Requests]
currently I'm using a reveal presentation with `theme: night` and adding 

`%%{init: {"theme": "dark" } }%%`
 
to each diagram. I'm not sure if this works for every diagram type--the mermaid website suggests that theming, for example, state machines has to be done via a different mechanism.

**Comment by cscheid** [comment]
The problem is that figuring out which themes should be "dark" and which should be "light" isn't obvious.

I do think what that we should have a yaml option to directly configure themes, though, that's a great idea.

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/1530
